services/scabbard/libscabbard/src/migrations/diesel/postgres/migrations/2022-03-29-160500-create-scabbard-store-tables/up.sql

```diff
@@ -52,7 +52,6 @@ CREATE TABLE IF NOT EXISTS consensus_2pc_action (
 
 CREATE TABLE IF NOT EXISTS consensus_2pc_update_context_action (
     action_id                 INTEGER PRIMARY KEY,
-    service_id                TEXT NOT NULL,
```

Comment:
It would be better to write these changes as new migrations (alter table, etc.) so we can update our environments.

Reply:
Updated